duckduckgo/iOS
# Manage Favorites from Overlay

Task/Issue URL: https://app.asana.com/0/414709148257752/1203858036570302/f

**Description**:
Allows Editing and Removing Favorites from the Favorites Overlay.

**Steps to test this PR**:
1. Fire up the app (Make sure you have a few 'Favorites' added)
2. Navigate to any website (Important ⚠️)
3. Tap the Omnibar
4. Long-Press on one of your favorites
5. Observe that the 'Edit' / 'Delete' popover appears
6. Test the functionality of both 'Edit' and 'Delete' from that view.

**Demo:**
![favorites](https://user-images.githubusercontent.com/1156669/225300863-120f06cb-7ead-4826-80c8-97b4878a4882.gif)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
